seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java

```diff
@@ -46,7 +56,7 @@ public String factoryIdentifier() {
     @Override
     public OptionRule optionRule() {
         return OptionRule.builder()
-                .required(URL, DATABASES, KEY_MEASUREMENT)
```

Comment:
please move `KEY_MEASUREMENT` to optional

Reply:
> please move `KEY_MEASUREMENT` to optional
Writing to config for a single table requires configuring KEY-MEASURE, but not for multi table writing. 
Reference kudu MultiTableSinkWriter
![image](https://github.com/apache/seatunnel/assets/33744252/db44860f-7b8e-48d1-9e8a-3bf11c71b39e)